### DISCUSSION TITLE
Hello, can I download the BIM file for this case locally?

### DISCUSSION THREAD

**Discussion by jh9636** [iTwin/community#48] [Q&A]
Hello, can I download the BIM file for this case locally?I would like to obtain this BIM file.
![~Y{4J26NIKEAZADY4(S8SE](https://github.com/iTwin/community/assets/164757029/3929ea3b-366e-499f-8aeb-37bcfec9f9c7)
https://www.itwinjs.org/sandboxes/iTwinPlatform/Cross-Probing

**Comment by Johannes-repos** [comment]
Well, there isn't an official download for this sample model but nothing would stop you to create it as one of your own using the My iTwins environment. Once you got that as an online iModel you can then use the Get iModel Baseline File Details API. The response has a download link you can use.
What are you trying to build with it?
